pkg/occlient/occlient.go

```diff
@@ -892,6 +899,21 @@ func (c *Client) GetClusterServiceClasses() ([]scv1beta1.ClusterServiceClass, er
 	return classList.Items, nil
 }
 
+func (c *Client) CreateServiceInstance(componentName string, componentType string, labels map[string]string, applicationName string, projectName string) error {
+	// Patching Template with ODO specific labels
+	labelPatch := fmt.Sprintf(`{"labels": {"app.kubernetes.io/component-name": "%s", "app.kubernetes.io/component-type": "%s", "app.kubernetes.io/name": "%s"}}`, componentName, componentType, applicationName)
+	_, err := c.templateClient.Templates("openshift").Patch(componentType, types.StrategicMergePatchType, []byte(labelPatch))
```

Comment:
won't this modify templates saved in `openshift` namespace?

Reply:
what is service is from some other broker than openshift template broker?

Reply:
I didn't get second question

Reply:
The function is called CreateServiceInstance, but is using OpenShift templates.
It is really confusing :-( 
There can be multiple different Service brokers in Service Catalog.